packages/@n8n/api-types/src/push/collaboration.ts

```diff
@@ -0,0 +1,16 @@
+import type { Iso8601DateTimeString } from '../types';
+
+export interface Collaborator {
+	userId: string;
```

Comment:
port over a `User` type into this package, and revert this back to `user: User` to avoid making unrelated changes in this PR.